.github/workflows/ci.yml

```
@@ -55,65 +60,117 @@ jobs:
-            github.rest.issues.createComment({
+            await github.rest.issues.createComment({
               owner: context.repo.owner,
               repo: context.repo.repo,
               issue_number: context.issue.number,
               body
             });
-        env:
-          BUILD_STATUS: ${{ needs.build-test.result }}
 
       - name: Slack notify
         if: ${{ always() && secrets.SLACK_WEBHOOK_URL != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)